api/src/main/java/org/openmrs/module/m2sysbiometrics/client/M2SysV105Client.java

```diff
@@ -162,4 +174,22 @@ private M2SysCaptureResponse scanDoubleFingers() {
                 getCloudScanrUrl() + M2SysBiometricsConstants.M2SYS_CAPTURE_ENDPOINT,
                 request, token, M2SysCaptureResponse.class);
     }
+
+    private FingerScanStatus checkIfFingerScanExists(M2SysCaptureResponse fingerScan) {
+        boolean existsLocally = false;
+        boolean existsNationally = false;
+        try {
+            existsLocally = localBioServerClient.isFingerScanExists(fingerScan);
+        } catch (APIException localClientAPIEx) {
+            LOG.error("Connection failure to local server.", localClientAPIEx);
```

Comment:
this should be an error that propagates - we should only proceed in case the national server is unreachable

Reply:
What about not defined URL?

Reply:
In that case do not query the national server at all